bouncer/__init__.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+import pyramid.config
+
+
+def settings():
+    """
+    Return the app's configuration settings as a dict.
+
+    Settings are read from environment variables and fall back to hardcoded
+    defaults if those variables aren't defined.
+
+    """
+    via_base_url = os.environ.get("VIA_BASE_URL", "https://via.hypothes.is")
+    if via_base_url.endswith("/"):
+        via_base_url = via_base_url[:-1]
+
+    return {
+        "elasticsearch_host": os.environ.get("ELASTICSEARCH_HOST",
+                                             "localhost"),
+        "elasticsearch_port": os.environ.get("ELASTICSEARCH_PORT",
+                                             "9200"),
+        "elasticsearch_index": os.environ.get("ELASTICSEARCH_INDEX",
+                                              "annotator"),
+        "chrome_extension_id": os.environ.get(
+            "CHROME_EXTENSION_ID",
+            "bjfhmglciegochdpefhhlphglcehbmek"),
+        "via_base_url": via_base_url,
+    }
+
+
+def app():
```

Comment:
Only used by `Makefile` and looks like this: `gunicorn --reload "bouncer:app()"`